terraform/vpn-user-data.sh

```diff
@@ -0,0 +1,183 @@
+#!/bin/bash
+set -e
+
+# OpenVPN Server Installation and Configuration with IAM Authentication
+# This script installs OpenVPN and configures it to authenticate users via IAM
+# Users must be in the '${vpn_iam_group}' IAM group to connect
+
+# Update system
+apt-get update
+apt-get upgrade -y
+
+# Install required packages
+apt-get install -y openvpn easy-rsa python3-pip jq awscli
+
+# Install boto3 for IAM authentication
+pip3 install boto3
+
+# Set up Easy-RSA for certificate generation
+make-cadir /etc/openvpn/easy-rsa
+cd /etc/openvpn/easy-rsa
+
+# Configure Easy-RSA
+cat > vars <<EOF
+set_var EASYRSA_REQ_COUNTRY    "AU"
+set_var EASYRSA_REQ_PROVINCE   "NSW"
+set_var EASYRSA_REQ_CITY       "Sydney"
+set_var EASYRSA_REQ_ORG        "OAF"
+set_var EASYRSA_REQ_EMAIL      "admin@openaustraliafoundation.org.au"
```

Comment:
Should this email be set to contact@oaf.org.au (or similar?)

```suggestion
set_var EASYRSA_REQ_EMAIL      "contact@oaf.org.au"
```